<launch>
    <node pkg="spirecv_ros" type="single_object_tracking" name="single_object_tracking" output="screen">
        <param name="uav_id" type="int" value="1" />
        <param name="camera_parameters" type="string" value="$(find spirecv_ros)/camera_params/calib_webcam_640x480.yaml" />
        <param name="local_saving_path" type="string" value="$(find spirecv_ros)/video" />
    </node>
</launch>
